src/_includes/install/php-extensions-template.md

```diff
@@ -6,3 +6,12 @@
 *  {{ item[0] }}
   {% endif %}
 {% endfor %}
+{% unless page.guide_version == '2.3' %}
+
+*  ext-sockets
+
+{% else %}
+
+{:.bs-callout-warning}
+If you install Magento by cloning from the [GitHub](https://github.com/magento/magento2) repository, you must have the [ext-sockets](https://github.com/php-amqplib/php-amqplib/blob/master/CHANGELOG.md#281---2018-11-13) installed on your instance.
```

Comment:
Like mentioned previously, I would remove this, it only applies for Magento 2.3.4 and 2.3.5. And who is installing Magento by cloning the git repo anyways? It's not really recommended for production deployments and now that those versions are outdated I don't see the point of mentioning this notice.

Just my 2 cents.